src/environments/environment.prod.ts

```diff
@@ -1,3 +1,14 @@
 export const environment = {
-  production: true
+  production: true,
+  firebase: 
+  {
```

Comment:
these values should never be stored in your repo. This is a security hole. Please read up on .env files. They are files you do not check in that have environment variables. CBT project uses one. Maybe take a look at that